UKHomeOffice/sgmr-service
# SPCR-461 Handle more than 10 people

### AC / Description of changes
API defaults to returning 10 people per GET call
Quick fix is to prompt it to return 100
Longer term design is being considered


### To Test

- create more than 10 people in your account
- create a voyage
- > on the select people for voyage you should see ALL your people
- select them all
- > on the manifest you should see ALL your people
- continue to employees
- > on select employees select ALL people as employees
- continue to CYA
- > you should see ALL your people in the people list
- > you should see ALL your employees in the employee list


### Notes

---

**Remember to merge to feature branches not `master`.**
